README.md

```diff
@@ -1,2 +1,16 @@
 # maven-properties-example
-An example project for working with system properties.
```

Comment:
you removed the wrong line.

Reply:
I replace it by "This is an example project for working with system properties."